package.json

```diff
@@ -157,5 +157,6 @@
     "webpack-cli": "^3.3.5",
     "xml2js": "^0.4.19"
   },
-  "private": true
+  "private": true,
+  "_id": "@cumulus/core@1.0.0"
```

Comment:
Why is this necessary?

Reply:
🤔  it's not.      

I'm entirely uncertain where that came from, but I'd expect it's an inadvertent add from an npm command.

Reply:
32443363a